src/geocoder.ts

```diff
@@ -60,9 +60,11 @@ function geocodeChunk(
   const rows = item.rows;
   item.tries++;
   //turn the rows into something smartystreets can read
-  const addressList = rows.map(function(row){
+  const geocodeInputsToRowsMap: { [key: number]: number } = {}
+  const geocodeInputs: { [key: string]: any }[] = []
+  rows.forEach(function(row, i){
     const result: { [col: string]: string } = {
-      city:     row[options.cityCol] || '',
+      city:     row[options.cityCol] || '', // this should be null not '' when empty...
```

Comment:
Please explain why `null` is more correct than `''` in this context. We're working with CSVs, where the distinction isn't always clear. Its there a SmartyStreets API issue here? We can change the type declarations to allow this if necessary.